developer-docs-site/docs/cli-tools/aptos-cli-tool/automated-install-aptos-cli.md

```diff
@@ -7,12 +7,20 @@ id: "automated-install-aptos-cli"
 
 The `aptos` tool is a command line interface (CLI) for developing on the Aptos blockchain, debugging Move contracts, and conducting node operations. This document describes how to install the `aptos` CLI tool using the automated install script.
 
+## Prerequisites
+
 First, ensure you have Python 3.6+ installed:
 ```
 $ python3 --version
 Python 3.9.13
 ```
-If it is not installed, you can find installation instructions [here](https://www.python.org/downloads/).
+If it is not installed, you can find installation instructions on [python.org](https://www.python.org/downloads/).
+
+Optionally, [install the Rust programming language and Cargo package manager](https://doc.rust-lang.org/cargo/getting-started/installation.html) for your operating system to enable automatic [updates](#update).
```

Comment:
This is not necessary, the automatic updates don't require Rust / cargo, the CLI can update itself without them (even without Python).

Reply:
Ah, but I thought you had already pointed out Python was needed. I have removed the Rust and Cargo reference here.

Reply:
Python is needed for the install script but the self updating feature of the CLI is standalone. Realistically anyone using the self update probably used the install script so they'll have Python, but it's not a requirement.